### PR TITLE
Add to cart

### PR DESCRIPTION
**Pull request by Aaron-Fink**
completed add to cart functionality. no tests are currently written. added a ternary statement to prevent components from loading without information.

### CODE REVIEW

**Comment by Aaron-Fink** [comment]
new commit makes tests pass

File: client/components/Overview/components/Sizes.jsx [hunk 1]
```diff
@@ -4,19 +4,31 @@ import React from 'react';
 import SizesRender from './SizesRender';
 
 //
-const Sizes = ({ currentStyle, setCurrentSize, setCurrentQuantity }) => {
+const Sizes = ({
```
Comment (greta-schock) [inline client/components/Overview/components/Sizes.jsx:7]: Low-key wondering why ESLint is throwing a fit about this. It did the same for me!

Comment (Aaron-Fink) [reply client/components/Overview/components/Sizes.jsx:7]: its the way it wants it formatted when the line gets too long

**Review by greta-schock — APPROVED** [review]
Looks good!

**Pull request merged**